python/src/spark_rapids_ml/regression.py

```diff
@@ -411,16 +424,32 @@ def __init__(
         dtype: str,
     ) -> None:
         super().__init__(dtype=dtype, n_cols=n_cols, coef_=coef_, intercept_=intercept_)
-        self.coef_ = coef_
-        self.intercept_ = intercept_
+        self._coef = coef_
+        self._intercept = intercept_
```

Comment:
Just curious why these are renamed?  Original intent was to just mirror the cuml attributes... and other classes follow the old naming scheme, so we should be consistent, if possible.

Reply:
Sorry, I didn't realize it. I just would like to hide the internal variable. Let me change them back.